test/utils/client.py

```diff
@@ -195,9 +195,13 @@ class TezosClient:
 
     backend: BackendInterface
 
+
     def __init__(self, backend) -> None:
         self.backend = backend
 
+    def finger_touch(self, x: int = 0, y: int = 0, delay: float = 0.5) -> None:
+        self.backend.finger_touch(x,y,delay)
+
```

Comment:
This code is unused

Reply:
Oh, that because the ChoiceList() needs a `BackendInterface` and not a `TezosClient`:
In https://github.com/trilitech/ledger-app-tezos-baking/blob/b169879f73b2575682a7debb7fe8eb67f0c1a9dd/test/utils/navigator.py#L117
The line should be:
```python
        self.layout_choice = ChoiceList(backend, firmware)
```
Sorry to have missed this one

Reply:
Got it. Changed.